lib/gis_robot_suite/descriptive_metadata_builder.rb

```diff
@@ -0,0 +1,626 @@
+# frozen_string_literal: true
+
+module GisRobotSuite
+  # Builds cocina descriptive metadata from ISO19139
+  class DescriptiveMetadataBuilder # rubocop:disable Metrics/ClassLength
+    def self.build(cocina_model:, bare_druid:, iso19139_ng:)
```

Comment:
Updated to make the arguments more explicit, since it sounds like that would help readability, @edsu and @ndushay. Although we have other builders in other repos passing unnamed arguments, this is the only builder class in this repo.